### PR TITLE
ruby: export a junit report in Ruby test targets if ROCK_TEST_LOG_DIR is set

### PR DESCRIPTION
**Pull request by doudou**
This is already what rock.core does for Ruby packages.

### CODE REVIEW

File: modules/RockRuby.cmake [hunk 1]
```diff
@@ -251,9 +252,22 @@ else()
             set(test_requires "${test_requires}require '${test_file}';")
         endforeach()
 
+        if (ROCK_TEST_LOG_DIR)
+            file(MAKE_DIRECTORY ${ROCK_TEST_LOG_DIR})
+            list(APPEND __minitest_args
+                --junit
```
Comment (2maz) [inline modules/RockRuby.cmake]: does that imply further dependencies, e.g., minitest-junit?

Comment (doudou) [reply modules/RockRuby.cmake]: It implies further dependencies if ROCK_TEST_LOG_DIR is set, and that you are running Ruby tests automatically using `autoproj test`.

The patch in the rock.core package set auto-adds this dependency for the same reason.

I believe it's a better alternative than simply having no test reporting. If you feel it isn't, I can hide that under a ROCK_RUBY_JRUBY_REPORTING flag as well.

Comment (2maz) [reply modules/RockRuby.cmake]: No problem, just did not have a look a the patch in rock-core. I would suggest, however, to mention/document this dependency here in the top section as well.

Comment (doudou) [reply modules/RockRuby.cmake]: Makes sense, will do.

Comment (doudou) [reply modules/RockRuby.cmake]: Done

**Pull request merged**